package/MDAnalysis/lib/formats/libmdaxdr.pyx

```diff
@@ -19,6 +19,32 @@
 # MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
+"""Low-level Gromacs XDR trajectory reading — :mod:`MDAnalysis.lib.formats.libmdaxdr`
+---------------------------------------------------------------------------------
+
+libmdaxdr contains the classes :mod:`XTCFile` and :mod:`TRRFile`. Both can be
+used to read and write frames from to XTC and TRR files. These classes are used
+internally by MDAnalysis in :mod:`MDAnalysis.coordinates.XTC` and
+:mod:`MDAnalysis.coordinates.TRR`. They behave similar to normal file objects.
+For example to calculate mean coordinates.
+
+.. code::python
+  with XTFile("trajectory.xtc") as xtc:
+      n_atoms = xtc.n_atoms
+      mean = np.zeros((n_atoms, 3))
+      # iterate over trajectory
+      for frame in xtc:
+          mean += frame.x
+
+The XTFile class can be usefull as a compressed storage format.
+
+Besides iteration XTCFile and TRRFile one can also seek to arbitrary frames
+using the `seek` method. This is provided by lazy generating a offset list for
+stored frames. The offset list is generated the first time `len` or `seek` is
+called.
```

Comment:
Good point – and proves my point that we need docs ;-).

Then the best thing to do is to add it to the XTC/TRR docs and then link from here to these docs. You can add anchors or "labels" with
```reST
.. _carthago-label:
```

and then link to it from anywhere in the docs with the [ref](http://www.sphinx-doc.org/en/1.4.9/markup/inline.html#ref-role) role:
```reST
Ceterum censeo :ref:`Carthago <carthago-label>` esse delendam.
```

If you put the label in the line before a heading then the heading text is substituted as the link text (just leave out the link text in ref).